src/test/java/com/monumental/services/integrationtest/MonumentRepositoryIntegrationTests.java

```diff
@@ -0,0 +1,300 @@
+
+    @Autowired
+    private MonumentRepository monumentRepository;
+
+    /* save Tests **/
+
+    /**
+     * Test method for integration testing MonumentRepository.save(record)
+     * Checks that the returned result from the save is not null, meaning it was saved correctly
+     */
+    @Test
+    public void testMonumentRepository_save() {
+        Monument monument = makeTestMonument();
+
+        monument = this.monumentRepository.save(monument);
+
+        assertNotNull(monument.getId());
+    }
+
+    /**
+     * Test method for integration testing MonumentRepository.saveAll(List<record>)
+     * Checks that the number of returned results is 3 and that each is not null, meaning they were all saved
+     * correctly
+     */
+    @Test
+    public void testMonumentRepository_saveAll() {
+        List<Monument> monuments = makeTestMonuments();
+
+        List<Monument> results = this.monumentRepository.saveAll(monuments);
+
+        assertEquals(3, results.size());
+
+        for (Monument result : results) {
+            assertNotNull(result.getId());
+        }
+    }
+
+    /* get Tests **/
+
+    /**
+     * Test method for integration testing MonumentRepository.get(record)
+     * First saves a record into the database then performs a get to verify the records are the same
+     */
+    @Test
+    public void testMonumentRepository_get_Single() {
+        Monument monument = makeTestMonument();
+
+        this.monumentRepository.save(monument);
+        Optional<Monument> updatedMonument = this.monumentRepository.findById(monument.getId());
+
+        assert(updatedMonument.isPresent());
+        assertEquals(updatedMonument.get().getTitle(), updatedMonument.get().getTitle());
+        assertEquals(updatedMonument.get().getArtist(), updatedMonument.get().getArtist());
+    }
+
+    /**
+     * Test method for integration testing MonumentRepository.getAll(List<record>)
+     * First saves 3 records into the database then performs a get for only 2 of the records
+     * Checks that only 2 records are returned and that they have the expected attributes
+     */
+    @Test
+    public void testMonumentRepository_getAll_ListPassed() {
+        ArrayList<Monument> monuments = makeTestMonuments();
+
+        this.monumentRepository.saveAll(monuments);
+
+        ArrayList<Integer> idsToGet = new ArrayList<>();
+        idsToGet.add(1);
+        idsToGet.add(3);
+
+        HashMap<Integer, Monument> monumentsById = new HashMap<>();
+        monumentsById.put(1, monuments.get(0));
+        monumentsById.put(3, monuments.get(2));
+
+        List<Monument> results = this.monumentRepository.findAllById(idsToGet);
+
+        assertEquals(2, results.size());
+
+        for (Monument result : results) {
+            Monument monument = monumentsById.get(result.getId());
+
+            assertEquals(monument.getTitle(), result.getTitle());
+            assertEquals(monument.getArtist(), result.getArtist());
+        }
+    }
+
```

Comment:
Can you fix up the comments and method names so they match the method we're calling? I.e. change `get` to `findById`, etc.

Reply:
See my posts above about the tests, I think this whole file can be scrapped honestly